components/wysiwyg.vue

```diff
@@ -648,6 +648,7 @@ export default {
     await import('vue-quill-editor').then(module => {
       this.QuillComponent = module.quillEditor
       this.$nextTick(() => {
+        this.$refs.myQuillEditor.quill.format('size', '1.00rem');
```

Comment:
@mahfouzhseineweev wasn't the issue due to a value of 1.00 ? why did you put it like this here ?
Also why using a fixed value here ?